.github/workflows/root-ci-config/build_utils.py

```diff
@@ -76,6 +76,14 @@ def print(self) -> None:
                 print(f"""\
 Grab the image:
 $ docker run {' '.join(self.docker_opts)} -it {self.image}
+Setup the Python virtual environment after checking out root and roottest (see below)
```

Comment:
Rather than giving this instructions, couldn't those be embedded in the 'right' place below, so that copy/pasting the output would work?

Reply:
Not easily, since this is something done in the workflow and not in the script, the entity logging the commands. Therefore, we somehow need to remind to perform the steps necessary to set up the right python environment.

Reply:
also, there was a mistake. The dependencies are installed when building the container, and not before the build.